pynucastro/networks/tests/_amrexastro_cxx_approx_reference/reaclib_rates.H

```diff
@@ -19,35 +22,6 @@ struct rate_derivs_t {
     Array1D<Real, NrateReaclib+1, NrateReaclib+NrateTabular> add_energy_rate;
 };
 
```

Comment:
The `struct tf_t` is removed and a header `tfactors.H` is created, and inserted by `#include<tfactors.H>`, also the `<partition_functions.H>` is included for testing purposes.

Reply:
yes, I needed to move it into its own header to avoid circular dependencies